ledger/catchupaccessor.go

```diff
@@ -298,11 +298,40 @@ func (c *catchpointCatchupAccessorImpl) ProcessStagingBalances(ctx context.Conte
 	if strings.HasPrefix(sectionName, "balances.") && strings.HasSuffix(sectionName, ".msgpack") {
 		return c.processStagingBalances(ctx, bytes, progress)
 	}
+	if sectionName == "stateProofVerificationData.msgpack" {
+		return c.processStagingStateProofVerificationData(ctx, bytes, progress)
+	}
 	// we want to allow undefined sections to support backward compatibility.
 	c.log.Warnf("CatchpointCatchupAccessorImpl::ProcessStagingBalances encountered unexpected section name '%s' of length %d, which would be ignored", sectionName, len(bytes))
 	return nil
 }
 
+// processStagingStateProofVerificationData deserialize the given bytes as a temporary staging state proof verification data
+func (c *catchpointCatchupAccessorImpl) processStagingStateProofVerificationData(_ context.Context, bytes []byte, _ *CatchpointCatchupAccessorProgress) (err error) {
+	var decodedData catchpointStateProofVerificationData
```

Comment:
Kind of strange I think, it's OK for the section to be missing if we catchup to an old catchpoint, and if it's missing in a new catchpoint the hash will fail. Kind of like how we don't test whether we've collected all chunks and return a special error. As I understand it, the content chunk (file header) is a special case since we want it to be the first thing we see, for accurate progress reports and such.

Reply:
@algorandskiy What are your thoughts? I'm think I should increase the catchpoint file's version and create the label differently when trying to verify it for the new version.

Reply:
@algorandskiy  Another thought - we could check whether we see stateProofVerificationData.msgpack in the catchpoint file, and if not, create the label for verification (in catchupaccessor) the old way. In new catchpoints such a file would be created and placed in the catchpoint even if no data exists in the stateProofVerification table, so this is a good way to differentiate securely between old catchpoints and new ones.

Reply:
I agree, introducing a new data type chunk requires a new catchpoint file version.

Reply:
Introduced versioning, resolving this discussion for now, please comment on the new label versioning.